docs/catalog/web/minify-web-assets.md

```diff
@@ -0,0 +1,34 @@
+---
+version: 1.0
+submitted_by: navveenb
+published_date: tbd
+category: web
+tags: 
+ - web-assets
+ - role:web-developer
+ - size:small
+---
+
+# Minify web assets
+
+## Description
+A web page comprises various assets, such as CSS, JavaScript, and third-party libraries like jQuery, Bootstrap etc. Minification removes unnecessary or redundant data without affecting how the resource is processed by the web browser.
```

Comment:
I would propose to:
* Add HTML, which can also be minified
* Remove the libraries, as these often come minified, and it's a different logical level as HTML/CSS/JS